ports/gumchugirl/port.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": 3,
+    "name": "gumchugirl.zip",
+    "items": [
+        "GUM CHU GIRL.sh",
+        "gumchugirl/"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "GUM CHU GIRL",
+        "porter": [
+            "MadShmupper"
+        ],
+        "desc": "Use your bubble gum to create temporary platforms to solve puzzles and traverse through the cosmic Starbase. Defeat the treacherous drill in order to save your solar system.",
+        "desc_md": null,
+        "inst": "Ready to Run",
+        "inst_md": null,
+        "genres": [
+            "platformer",
+            "puzzle",
+            "strategy"
+        ],
+        "image": {
+            "screenshot": "screenshot.png",
```

Comment:
Why are the image fields filled out?
Did you take a installed port.json file?

Reply:
Fixed it..ty